assets/templates/signup.html

```diff
@@ -30,48 +30,36 @@
 </head>
 
 <body>
-  <div class="panel-wrapper">
-    <section class="main-panel">
-
-      <div class="main-panel__head">
+  <main class="panel-wrapper">
+    <article class="main-panel">
+      <header>
         <img src="img/logo-doppler.svg" class="logo-doppler" alt="Doppler">
-        <p class="head-signin">¿Ya tienes una cuenta? <a href="#" class="link-green" target="_blank">INGRESA</a></p>
-      </div>
+        <p class="content-signin">¿Ya tienes una cuenta? <a href="#" class="link-green" target="_blank">INGRESA</a></p>
+      </header>
+      <h5>REGÍSTRATE</h5>
```

Comment:
I agree with Andres regarding uppercase texts. It would be best to add text-transform in styles, otherwise when we translate into language texts, we will need to be careful. I think that it can lead no necessary extra work. What do you think?

Reply:
Okey, there is consensus!

Reply:
@FromDoppler/doppler  What do you guys think?